### PR TITLE
[notebook] 2m timeout

### PR DESCRIPTION
**Pull request by akotlar**

### CODE REVIEW

File: notebook/notebook/notebook.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 routes = web.RouteTableDef()
 
 # Must be int for Kubernetes V1 api timeout_seconds property
-KUBERNETES_TIMEOUT_IN_SECONDS = float(os.environ.get('KUBERNETES_TIMEOUT_IN_SECONDS', 5))
+KUBERNETES_TIMEOUT_IN_SECONDS = float(os.environ.get('KUBERNETES_TIMEOUT_IN_SECONDS', 60))
```
Comment (cseed) [inline notebook/notebook/notebook.py]: Don't change two things at once, please.  If we want the additional logging, let's put that in first and study what happened, make the timeout change, and compare.

File: notebook/notebook/notebook.py [hunk 2]
```diff
@@ -241,12 +242,12 @@ def notebook_status_from_pod(pod):
                         cookies=cookies) as session:
                     async with session.get(ready_url) as resp:
                         if resp.status >= 200 and resp.status < 300:
-                            log.info(f'GET on jupyter pod {pod_name} succeeded: {resp}')
+                            log.info(f'notebook_status_from_notebook: GET on jupyter pod {pod_name} succeeded: {resp}')
```
Comment (cseed) [inline notebook/notebook/notebook.py]: The logging format puts the function name in the log already.

File: notebook/notebook/notebook.py [hunk 3]
```diff
@@ -241,12 +242,12 @@ def notebook_status_from_pod(pod):
...
                             status['state'] = 'Ready'
                         else:
-                            log.info(f'GET on jupyter pod {pod_name} failed: {resp}')
+                            log.error(f'notebook_status_from_notebook: GET on jupyter pod {pod_name} failed: {resp}')
```
Comment (cseed) [inline notebook/notebook/notebook.py]: These are not errors.  We fully expect the probe attempts to fail while the notebook is coming up.  Errors are things that human should ultimately have to investigate.

File: notebook/notebook/notebook.py [hunk 4]
```diff
@@ -389,9 +390,10 @@ def notebook_status_from_pod(pod):
             new_status = await notebook_status_from_notebook(k8s, service, headers, cookies, notebook)
             changed = await update_notebook_return_changed(dbpool, user_id, notebook, new_status)
             if changed:
+                log.info(f"_wait_websocket: pod {notebook['pod_name']} status changed: old status: {notebook['state']}, new status: {new_status['state']}")
```
Comment (cseed) [inline notebook/notebook/notebook.py]: I prefer `status changed: old => new`.

File: notebook/notebook/notebook.py [hunk 5]
```diff
@@ -389,9 +390,10 @@ def notebook_status_from_pod(pod):
             new_status = await notebook_status_from_notebook(k8s, service, headers, cookies, notebook)
             changed = await update_notebook_return_changed(dbpool, user_id, notebook, new_status)
             if changed:
+                log.info(f"pod {notebook['pod_name']} status changed: {notebook['state']} => {new_status['state']}")
                 break
         except Exception:  # pylint: disable=broad-except
-            log.exception('while updating status in /wait')
+            log.exception(f"/wait: error while check/update status for pod: {notebook['pod_name']}")
```
Comment (cseed) [inline notebook/notebook/notebook.py]: Nit: "error while check/update status" isn't grammatical.

Comment (akotlar) [reply notebook/notebook/notebook.py]: doen

File: notebook/notebook/notebook.py [hunk 6]
```diff
@@ -389,9 +390,10 @@ def notebook_status_from_pod(pod):
             new_status = await notebook_status_from_notebook(k8s, service, headers, cookies, notebook)
             changed = await update_notebook_return_changed(dbpool, user_id, notebook, new_status)
             if changed:
+                log.info(f"pod {notebook['pod_name']} status changed: {notebook['state']} => {new_status['state']}")
                 break
         except Exception:  # pylint: disable=broad-except
-            log.exception('while updating status in /wait')
+            log.exception(f"/wait: error while checking or updating status for pod: {notebook['pod_name']}")
```
Comment (cseed) [inline notebook/notebook/notebook.py]: Sorry to be pedantic here, but I don't understand what this message means anymore.  What's checking vs updating?  We're not using the term check anywhere else in this code.  If you check on the status of the pod, your understanding is still updated (that is, it is more up to date) even if the status hasn't changed.

Comment (akotlar) [reply notebook/notebook/notebook.py]: Sure. Checking was a reference to "notebook_status_from_notebook" which fetched the current status of the pod. 

I reverted it, let me know if you'd like something else.

**Pull request merged**